package.json

```diff
@@ -13,10 +13,15 @@
   },
   "license": "MIT",
   "devDependencies": {
+    "@types/plotly.js-dist-min": "^2.3.4",
     "tailwindcss": "^3.0.24",
     "ts-loader": "^9.3.0",
     "typescript": "^5.3.3",
     "webpack": "^5.76.0",
     "webpack-cli": "^4.9.2"
+  },
+  "dependencies": {
+    "plotly.js-dist-min": "^2.33.0",
+    "xlsx": "^0.18.6"
```

Comment:
Getting this error when I try to build:

```
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/xlsx/-/xlsx-0.18.6.tgz - Not found
npm ERR! 404 
npm ERR! 404  'xlsx@https://registry.npmjs.org/xlsx/-/xlsx-0.18.6.tgz' is not in this registry.
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

I think it would be ideal to remove this library anyway though